mysite/EStore/models/order.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+from .customer import Customers
+
+class Orders(models.Model):
+    customer = models.ForeignKey(to = Customers, on_delete = models.CASCADE)
```

Comment:
should use string,
`customer = models.ForeignKey(to = "EStore.Customers", on_delete = models.CASCADE)`